modules/persistent-storage-local-tolerations.adoc

```diff
@@ -52,4 +52,14 @@ To configure local volumes for scheduling on tainted nodes:
 <4> The volume mode, either `Filesystem` or `Block`, defining the type of the local volumes.
 <5> The path containing a list of local storage devices to choose from.
 
+. Optional: To create local persistent volumes (PVs) on only tainted nodes, modify the YAML file and add the `LocalVolume` spec, as shown in the following example:
```

Comment:
Since you're not referring to persistent volumes again in this module, you don't need `(PVs)`.

Reply:
Acked